test/topology/test_mutation_schema_change.py

```diff
@@ -0,0 +1,151 @@
+
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test repro of failure to store mutation with schema change and a server down
```

Comment:
The better description of the problem would be:
Reproducer for a failure during lwt operation due to missing of a column mapping in schema history table.

And please include the ref to the fix here 15ebd5907177ba8709bc40ad2fa9ae773a9cf674 which also have a longer description of the problem.